index.js

```diff
@@ -9,9 +9,23 @@ const boxen = require("boxen");
 const { exec } = require('node:child_process')
 const inquirer = require('inquirer');
 const {sync,listAll,initClient} = require('./sync');
+const https = require('https');
 
 let amplifyConfig, amplifyMeta;
 let backendType = 's3';
+
+const agent = new https.Agent({
+    maxSockets: 300,
```

Comment:
defaultMaxSockets is Infinity,we do not need to set a large number. normally we set this up to lower the max connections.```suggestion
    maxSockets: 300,
```